sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -144,7 +143,7 @@ def deduct_fees(self, request, cons_time, ledger_id, seq_no, txn):
                 txn = reqToTxn(txn)
                 self.token_ledger.append_txns_metadata([txn], txn_time=cons_time)
                 _, txns = self.token_ledger.appendTxns([TokenReqHandler.transform_txn_for_ledger(txn)])
-                self.updateState(txns)
+                super().updateState(txns)
```

Comment:
I think this is fine to call `self.updateState` here. What is not fine, is that `StaticFeesReqHandler` overrides `updateState` method hiding Indy-Node's behavior!
A correct is something like this:
```
    def updateState(self, txns, isCommitted=False):
        for txn in txns:
            self._update_state_with_single_txn(txn, is_committed=isCommitted)
        super().updateState(txns, isCommitted)
```

Reply:
Or we can put super().updateState calling into _update_state_with_single_txn method in 'else' section.